### PR TITLE
OADP-4872: Remove OADP 1.3 release notes from OCP 4.16

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-4872](https://issues.redhat.com/browse/OADP-4872)

The contents for [modules/oadp-verifying-upgrade-1-4-0.adoc](https://github.com/openshift/openshift-docs/blob/3944481c80346f3fcd3fbbd0a5ac37af90428cf9/modules/oadp-verifying-upgrade-1-4-0.adoc) come from this PR - https://github.com/openshift/openshift-docs/pull/68590 and this original PR - https://github.com/openshift/openshift-docs/pull/67404 that was not merged

### Version(s):

* OCP 4.16 +

### Link to docs preview:

* [OADP 1.4.z release notes](https://81876--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.html)
### QE review:

- [ ] QE has approved this change.

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc [hunk 1]
```diff
@@ -12,10 +12,13 @@ The release notes for {oadp-first} describe new features and enhancements, depre
 include::modules/oadp-1-4-0-release-notes.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-dpa-configuration-1-4-0.adoc[leveloffset=+3]
 include::modules/oadp-upgrading-oadp-operator-1-4-0.adoc[leveloffset=+3]
+
 [role="_additional-resources"]
 .Additional resources
+* link:https://access.redhat.com/articles/5456281[{oadp-first} for Data Protection FAQs]
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc:18]:
```suggestion
* link:https://access.redhat.com/articles/5456281[{oadp-first} FAQs]
```

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 2]
```diff
@@ -12,7 +12,12 @@ Because all taken backup remains until expired, also check the time to live (TTL
 
 [NOTE]
 ====
-There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:../../../backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1.adoc#known-issues_oadp-release-notes[Known issues].
+There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation.
+
+You can create many snapshots of the same persistent volume claim (PVC) but cannot schedule periodic creation of snapshots:
+
+* For CephFS, you can create up to 100 snapshots per PVC.
+* For RADOS Block Device (RBD), you can create up to 512 snapshots for each PVC. (link:https://issues.redhat.com/browse/OADP-804[*OADP-804*]) and (link:https://issues.redhat.com/browse/OADP-975[*OADP-975*])
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:20]: As we would cut 1.1 release notes, i have moved this section over to replace the link

Do we need to keep this?

File: backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc [hunk 3]
```diff
@@ -9,19 +9,22 @@ toc::[]
...
+====
+
 include::modules/oadp-1-4-0-release-notes.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-dpa-configuration-1-4-0.adoc[leveloffset=+3]
 include::modules/oadp-upgrading-oadp-operator-1-4-0.adoc[leveloffset=+3]
+
 [role="_additional-resources"]
 .Additional resources
-* xref:../../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Updating installed Operators]
+* link:https://access.redhat.com/articles/5456281[{oadp-first} FAQs]
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc:23]:
```suggestion
* xref:../../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Updating installed Operators]
```

**Comment by anarnold97** [comment]
will need to rebase, after [OADP 1.4.1 release notes](https://github.com/openshift/openshift-docs/pull/81739) are merged

**Review by weshayutin — APPROVED** [review]
/LGTM

**Review by rayfordj — APPROVED** [review]
/lgtm

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc (hunk 2 above)
Comment (anarnold97) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:20]: @amastbau - please can I flag this question to you? thanks

Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:20]:
```suggestion
```

https://redhat-internal.slack.com/archives/C0144ECKUJ0/p1726584003382159?thread_ts=1726582472.065089&cid=C0144ECKUJ0

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 4]
```diff
@@ -12,7 +12,6 @@ Because all taken backup remains until expired, also check the time to live (TTL
 
 [NOTE]
 ====
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:14]:
```suggestion
```

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 5]
```diff
@@ -10,9 +10,6 @@ toc::[]
 Frequent backups might consume storage on the backup storage location. Check the frequency of backups, retention time, and the amount of data of the persistent volumes (PVs) if using non-local backups, for example, S3 buckets.
 Because all taken backup remains until expired, also check the time to live (TTL) setting of the schedule.
 
-[NOTE]
-====
-There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:../../../backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1.adoc#known-issues_oadp-release-notes[Known issues].
 ====
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:13]:
```suggestion
```

**Review by rayfordj — APPROVED** [review]
/lgtm

**Review by rayfordj — APPROVED** [review]
/lgtm

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by anarnold97** [comment]
@GroceryBoyJr - thanks

**Comment by GroceryBoyJr** [comment]
@anarnold97 Be aware of the link brought forward from the 1.2 release note, the link to the vmware-tanzu github repo -there should be approval by PX (Eric Rich) somewhere in https://github.com/openshift/openshift-docs/pull/68590. I would mention that in your comment 0.

**Comment by stevsmit** [comment]
LGTM!

**Pull request merged**